README.rst

```diff
@@ -38,6 +38,10 @@
 .. image:: https://img.shields.io/github/license/SwissDataScienceCenter/renku-python.svg
    :target: https://github.com/SwissDataScienceCenter/renku-python/blob/master/LICENSE
 
+.. image:: https://pullreminders.com/badge.svg
+   :target: https://pullreminders.com?ref=badge
+   :alt: Pull reminders
```

Comment:
```suggestion
   :alt: Pull reminders
   :align: right
```

Reply:
@ableuler do we want to keep it right as in the other repos?

Reply:
I tried, but rst insisted on putting it onto a new line, so I thought left-justified looks better.